pgx/src/io_intf.ml

```diff
@@ -25,7 +27,8 @@ module type S = sig
   module Sequencer : sig
     type 'a monad = 'a t
     type 'a t
-
+    val sexp_of_t : 'a t -> Sexplib0.Sexp.t
+    val t_of_sexp : Sexplib0.Sexp.t -> 'a t
```

Comment:
Same thing here. I'm pretty sure sexping a Sequencer doesn't make any sense?

Reply:
Agreed. Most of this was a result of trying to fix compilation errors from whereever we were using `@@deriving sexp`. I'll change them all to use `sexp_of` and remove the `t_of_sexp` converters